blackbox/docs/sql/reference/select.txt

```diff
@@ -242,6 +243,34 @@ resulting row of a group by clause.
    Grouping can only be applied on indexed fields. For more information, please
    refer to :ref:`sql_ddl_index_off`.
 
+.. _sql_reference_union:
+
+UNION ALL Clause
+................
+
+The UNION ALL operator computes the set union of the rows returned by the
+involved SELECT statements. The two SELECT statements that represent
+the direct operands of the UNION ALL must produce the same number of columns,
+and corresponding columns must be of compatible data types. The result of
+UNION ALL may contain duplicate rows.
+
+::
+
+    UNION ALL select
```

Comment:
to be more generic I would rename the `select`  -> `query_specification` (see https://docs.microsoft.com/en-us/sql/t-sql/language-elements/set-operators-union-transact-sql)